hub/powertoys/run.md

```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+| Windows services                                                                     | `!`                       | `! alg` to search for the 'Application Layer Gateway' service to be started or stopped.                                                                                                                                                                                                              |
+| Shell command                                                                        | `>`                       | `> ping localhost` to do a ping query.                                                                                                                                                                                                                                                               |
+| Time and date                                                                        | `(`                       | `( time and date` shows the current time and date in different formats.<br />`( calendar week::04/01/2022` shows the calendar week for the date '04/01/2022'.                                                                                                                                        |
+| Time zones                                                                           | `&`                       | `& Newfoundland` shows the current time in the time zone of Newfoundland.                                                                                                                                                                                                                            |
+| Unit converter                                                                       | `%%`                      | `%% 10 ft in m` to calculate the number of meters in 10 feet.                                                                                                                                                                                                                                        |
+| URIs                                                                                 | `//`                      | `//` to launch your default browser.<br />`// docs.microsoft.com` to have your default browser go to https://docs.microsoft.com.<br />`mailto:` and `ms-settings:` links are supported.                                                                                                              |
+| Visual Studio Code<br />_previously opened workspaces, remote machines and containers_ | `{`                     | `{ powertoys` to search for workspaces that contain 'powertoys' in their paths.                                                                                                                                                                                                                      |
```

Comment:
If we keep description it should be in brakets I think.

Reply:
Moved these words into the example text.